File: src/main/java/ac/dnd/server/admission/infrastructure/web/dto/request/ApplicantStatusCheckRequest.java
package ac.dnd.server.admission.infrastructure.web.dto.request;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;

public record ApplicantStatusCheckRequest(
	@NotBlank
	String name,
	@Email
	@NotBlank
	String email
) {
}

File: src/main/kotlin/ac/dnd/server/admission/infrastructure/web/dto/request/ApplicantStatusCheckRequest.kt
package ac.dnd.server.admission.infrastructure.web.dto.request

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

data class ApplicantStatusCheckRequest(
    @field:NotBlank
    val name: String,
    
    @field:Email
    @field:NotBlank
    val email: String
)